### DISCUSSION TITLE
Plugin advice

### DISCUSSION THREAD

**Discussion by imbirWIthSugar** [Pryaxis/TShock#2291] [Q&A]
Tell me what plugins there are with functions such as: Displaying text on the screen, creating your own progress bar and economy.

[Converted from #2290]

**Comment by bartico6** [comment]
>your own progress bar  

none public
> displaying text on the screen  

none public
> economy  

sEconomy.

**Reply by moisterrific** [reply]
so basically unavailable, unavailable, and unusable lol

I don’t reckon ever finding a public SEconomy fork that worked.

**Reply by Kojirremer** [reply]
https://github.com/Quinci135/SEconomy 👀

**Comment by Kojirremer** [comment]
Your request is pretty vague to begin with. Create text on the screen? What text? Stats? Information about the terrain? Combat text? Your own message? Also, where on the screen? The chat? Smack middle on the screen, over the player, on the side, middle top...? Same with progress bar. For what? Do you mean modify invasion and/or boss bars? Add a custom bar for some unknown quest?
As for the economy thing, as quake said the only public and functional one available (AFAIK) is SEconomy, which also "creates text on the screen", specifically on the right side of the screen, under the minimap.
